src/components/cards/Treemap.vue

```diff
@@ -275,6 +274,12 @@ export default class Treemap extends TreemapProps {
 
   // Callbacks for our tree
   selectNode(n: d3.HierarchyRectangularNode<TreemapDatumType>): void {
+    // Avoid selecting falsey nodes
+    if (!n) {
```

Comment:
Uncertain if this is a case that's even possible to occur; if it is, we should also make the "selectNode" call signature to specify n: null | d3.Heirarchy...DatumType>

Reply:
Will add the new type signature, also before merging I am going to add the centering to the cards text.